docs/pages/workflows/get-started.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Get Started
+description: Learn how to use Workflows to automate your development and release processes.
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Callout } from '~/ui/components/Callout';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+<Callout type="info">
+  Workflows are currently in preview. We may introduce breaking changes in the coming weeks. Got
+  feedback? Send us an email at workflows@expo.dev.
+</Callout>
+
+Builds, submissions, updates and more are all a part of delivering your app to users. Workflows consist of a sequence of jobs, which help you and your team get things done. You can do things like building your project, run end-to-end tests, submit that build to the app stores, then run custom scripts after the submission is complete. Since each job can have prerequisites and conditionals, you can automate you and your team's release process.
+
+<Collapsible summary="How do workflows compare to other CI services?">
+
+Workflows on EAS are designed to help you and your team release your app. It comes preconfigured with pre-packaged job types that help you build, submit, update, run Maestro tests, and more. It all runs on EAS, so you'll only have to manage one set of YAML files, and all of the artifacts from your job runs will appear on expo.dev.
```

Comment:
```suggestion
Workflows on EAS are designed to help you and your team release your app. It comes preconfigured with pre-packaged job types that can build, submit, update, run Maestro tests, and more. All job types run on EAS, so you'll only have to manage one set of YAML files, and all the artifacts from your job runs will appear on [expo.dev](https://expo.dev/).
```

Let's make it a bit concise. It'll be an improvement to define pre-packaged jobs in the intro maybe.